src/api/Gemfile.lock

```diff
@@ -116,11 +116,11 @@ GEM
     faker (1.7.3)
       i18n (~> 0.5)
     feature (1.4.0)
+    ffi (1.9.17)
```

Comment:
If this is cosmetic... why did you added to the commit? I prefer to checkout the file in this case.